.travis.yml

```diff
@@ -10,18 +10,19 @@ branches:
   - devel
 env:
   global:
-  - IMAGE_REPO=dddecaf/wsgi-base
-  - IMAGE_TAG=${TRAVIS_BRANCH}
-  - IMAGE_HASH_TAG=${TRAVIS_COMMIT::12}
```

Comment:
I think having a hash-based image tag is desirable. Otherwise we overwrite our docker registry and lost the ability to roll back the base image, no? Could be `wsgi-base:alpine-${HASH}`, for example.

Reply:
Hmm, I have a hard time envisioning a scenario where we need that ability. Have we used it in the past?

Reply:
Hmm, I have a hard time envisioning a scenario where we need that ability. Have we used it in the past? Doesn't cost much to add it though so I will.